callbacks.go

```diff
@@ -9,6 +9,8 @@ import (
 	mesos "github.com/mesos/mesos-go/mesosproto"
 	"github.com/Nitro/sidecar-executor/container"
 	"github.com/relistan/go-director"
+	"github.com/fsouza/go-dockerclient"
+	"strconv"
```

Comment:
You should run this file through the `goimports` tool which will fix these imports and any other formatting issues (the imports are not properly ordered right now).